spec/models/claim/matching_attribute_finder_spec.rb

```diff
@@ -37,6 +38,12 @@
       expect(matching_claims).to be_empty
     end
 
+    it "does not include claims that match, but have a different academic year" do
+      create(:claim, :submitted, teacher_reference_number: source_claim.teacher_reference_number, academic_year: AcademicYear.new("2020"))
```

Comment:
I think we need to lock the policy to `StudentLoans` here.

Reply:
Good call 👍